commands/issue/issue_clone.go

```diff
@@ -37,11 +36,19 @@ func issueSubCloneCommand() *cli.Command {
 
 // Setting as a variable so that it can be overridden with a mkdir for tests
 var CloneRepositoryOnBranch = func(ctx *runtime.Context, dir, branch, gitPrefix string) error {
-	cloneUrl := path.Join(gitPrefix, filepath.Base(dir)+".git")
+	parser, err := GitConfig().Open(path.Join(dir, ".git/config"))
```

Comment:
This could be done outside of this function, and simply pass in the cloneURL.  As the only reason for the var is so that we don't have to actually clone repositories in tests.